frontend/src/app/my-links/page.tsx

```
@@ -0,0 +1,196 @@
+                <span>아직 생성된 폴더가 없습니다</span>
+              </div>
+            </div>
+          )}
+
+          {/* ── 폴더 카드 목록 (백엔드 데이터 반복 렌더링) ── */}
+          {folders?.map((folder) => (
+            <div
+              key={folder.memberFolderId}
+              onClick={() => setSelectedFolderId(folder.memberFolderId)}
+              className="bg-white dark:bg-card-dark rounded-lg p-2.5 border border-gray-100 dark:border-gray-800 hover:shadow-sm hover:border-purple-200 dark:hover:border-purple-800 transition-all group cursor-pointer h-[90px] flex flex-col justify-between"
+            >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)